src/main/java/com/agency04/sbss/pizza/rest/FirstController.java

```diff
@@ -0,0 +1,21 @@
+package com.agency04.sbss.pizza.rest;
+
+import com.agency04.sbss.pizza.model.pizzaPojos.Vegeteriana;
+import com.agency04.sbss.pizza.service.PizzaDeliveryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class FirstController {
```

Comment:
U zadatku se nije trazilo da se napravi Controller, makni ga. Rade ovaj dio zadatka "Make sure the app works as before, and the PizzeriaService used depends on which Bean has @Primary" napravit u main metodi